lib/libc/common/Kconfig

```diff
@@ -58,6 +58,7 @@ config COMMON_LIBC_MALLOC_ARENA_SIZE
 	int "Size of the common C library malloc arena"
 	depends on COMMON_LIBC_MALLOC
 	default 0 if MINIMAL_LIBC
+	default 2048 if PICOLIBC
```

Comment:
nack, this works fine for every other system and -1 is what people expect, the esp memory should be the one that changes

Reply:
> nack, this works fine for every other system and -1 is what people expect, the esp memory should be the one that changes

I agree! I put this PR only to test the impact. I'll update soon. And thanks for taking a look @thedjnK